drivers/spi/spi_nrfx_spim.c

```diff
@@ -634,6 +632,7 @@ static int spi_nrfx_init(const struct device *dev)
 			SPI_NRFX_SPIM_EXTENDED_CONFIG(idx)		       \
 		},							       \
 		.irq_connect = irq_connect##idx,			       \
+		.max_chunk_len = BIT_MASK(SPIM_PROP(idx, easydma_maxcnt_bits)),\
```

Comment:
I don't understand what is better in introduction of this DT property over using Nordic MDK define directly.
@anangl Can you clarify to us what is right approach in drivers?

Reply:
See the rationale here https://github.com/zephyrproject-rtos/zephyr/pull/46726 (was partially merged in https://github.com/zephyrproject-rtos/zephyr/pull/46984)

Reply:
So fare idea for shims was to us the nrfx as the base. the Nordic nrfx team provides these code mainly for zephyr-rtos drivers. What I think is that we should first discus with nrfx developer what is needed for use and what they can rework and then introduce new approach to the Nordic drivers.
Otherwise we will looping in a wheel reinventing.

Coping by hand things which were generated to the MDK/nrfx to a DTS looks like not a good idea to me.

Reply:
nrfx, as it is, is far from being Zephyr friendly when it comes to drivers instantiation. Just compare what we have w.r.t. any other vendor.

Reply:
One may be a silly question. Who will be responsible for maintaining this? The previous approach required 0 maintenance and this one requires rewriting it by hand.

I really don't like that way of doing things / introducing changes. Instead of a bottom-up approach, I feel like you're enforcing a top-down strategy that introduces unnecessary overhead. I would suggest starting a discussion on how to make nrfx more "zephyr-friendly". What are your thoughts? @gmarull @anangl @nika-nordic @carlescufi